airbyte-integrations/connectors/source-salesloft/source_salesloft/source.py

```diff
@@ -226,6 +226,52 @@ def path(self, **kwargs) -> str:
         return "groups"
 
 
+class CustomFields(IncrementalSalesloftStream):
+    created_at_field = "updated_at"
+
+    def path(self, **kwargs) -> str:
+        return "custom_fields"
```

Comment:
This stream can't be incremental because it doesn't allow to filter records based on `updated_at` parameter.